cranelift/codegen/src/machinst/lower.rs

```diff
@@ -122,18 +122,18 @@ pub trait LowerBackend {
     /// For a branch, this function should not generate the actual branch
     /// instruction. However, it must force any values it needs for the branch
     /// edge (block-param actuals) into registers, because the actual branch
-    /// generation (`lower_branch_group()`) happens *after* any possible merged
+    /// generation (`lower_branch()`) happens *after* any possible merged
```

Comment:
The `CodegenResult` return type was previously self-documenting, but the `Option` now is a little ambiguous. Can we add a note here that returning `None` indicates that a lowering is missing for the given instruction (and likewise below for `lower_branch`)?

Reply:
Makes sense; I've added this now.